.docs/user-guide/docker-plugins.md

```diff
@@ -278,6 +278,52 @@ Environment Variable | Description | Default | Required
 `GCEPD_TAG` | Only use volumes that are tagged with a label | |
 `GCEPD_ZONE` | GCE Availability Zone | |
 
+## Openstack
+REX-Ray ships with plug-ins for OPenstack as well.
+
+### Cinder volume
+The CINNDER plug-in can be installed with the following command:
```

Comment:
Hi @sebglon,

Please correct all instances of `CINDER` so they are `Cinder` as it is the [official casing](https://wiki.openstack.org/wiki/Cinder):

![image](https://cloud.githubusercontent.com/assets/101085/26170785/37266a16-3b08-11e7-9b6d-640049091a4c.png)